File: src/test/fixtures/python/coherence/static_god_class.py
# An all-static class is a namespace of functions, not an instance responsibility aggregate.
class StaticUtilities:
    @staticmethod
    def one(value: int) -> str: return ""
    @staticmethod
    def two(value: str) -> float: return 0.0
    @staticmethod
    def three(value: float) -> bytes: return b""
    @staticmethod
    def four(value: bytes) -> list: return []
    @staticmethod
    def five(value: list) -> dict: return {}
    @staticmethod
    def six(value: dict) -> tuple: return ()
    @staticmethod
    def seven(value: tuple) -> set: return set()
    @staticmethod
    def eight(value: set) -> bool: return False
    @staticmethod
    def nine(value: bool) -> object: return object()
    @staticmethod
    def ten(value: object) -> complex: return 0j
    @staticmethod
    def eleven(value: complex) -> range: return range(0)
    @staticmethod
    def twelve(value: range) -> memoryview: return memoryview(b"")
    @staticmethod
    def thirteen(value: memoryview) -> bytearray: return bytearray()
    @staticmethod
    def fourteen(value: bytearray) -> frozenset: return frozenset()
    @staticmethod
    def fifteen(value: frozenset) -> int: return 0
    @staticmethod
    def sixteen(value: int) -> str: return ""

File: src/test/fixtures/typescript/coherence/staticGodClass.ts
// An all-static class is a namespace of functions, not an instance responsibility aggregate.
export class StaticUtilities {
    static one(value: number): string { return ""; }
    static two(value: string): number { return 0; }
    static three(value: boolean): Uint8Array { return new Uint8Array(); }
    static four(value: Uint8Array): string[] { return []; }
    static five(value: string[]): Record<string, unknown> { return {}; }
    static six(value: Record<string, unknown>): [number] { return [0]; }
    static seven(value: [number]): Set<number> { return new Set(); }
    static eight(value: Set<number>): Map<string, number> { return new Map(); }
    static nine(value: Map<string, number>): Date { return new Date(); }
    static ten(value: Date): RegExp { return /x/; }
    static eleven(value: RegExp): Error { return new Error(); }
    static twelve(value: Error): Promise<void> { return Promise.resolve(); }
    static thirteen(value: Promise<void>): ArrayBuffer { return new ArrayBuffer(0); }
    static fourteen(value: ArrayBuffer): DataView { return new DataView(value); }
    static fifteen(value: DataView): URL { return new URL("https://example.com"); }
    static sixteen(value: URL): number { return 0; }
}
